plugins/source/aws/resources/services/ssoadmin/customer_managed_policies.go

```diff
@@ -0,0 +1,59 @@
+package ssoadmin
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin"
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func customerMangedPolices() *schema.Table {
```

Comment:
function name is misspelled for Managed (Manged) and Policies (Polices)

Reply:
fixed in https://github.com/cloudquery/cloudquery/pull/10985/commits/00c529da2c80c93084346378b9026edd554ee68d